app/controllers/post_controller.rb

```diff
@@ -1,17 +1,24 @@
 require 'uri'
+
 ##
 # The controller responsible for dealing with views related to SSE website posts
 class PostController < BasePostEditorController
   # GET post/list
   def list
-    @posts = GithubService.get_all_posts(session[:access_token])
+    PostImageManager.instance.clear
+    @pr_posts = GithubService.get_all_posts_in_pr_for_user(session[:access_token])
+    @posts = GithubService.get_all_posts(session[:access_token]).select do | post |
+      found_post = @pr_posts.find { |x| x.title == post.title }
+      post if !found_post
+    end
+    @posts.compact!
   end
   
   # GET post/edit
```

Comment:
What is the `ref` for a post? I think the `title` is self-explanatory, but not sure about the `ref`. This may need additional documentation.

Reply:
I added some comments explaining below, but think of it as an object that points to a commit. In this case the parameter is a sha which points to the head of the base branch in the open PR.